source/introduction.txt

```diff
@@ -268,18 +266,18 @@ Prebuilt Applications
 ~~~~~~~~~~~~~~~~~~~~~
 
 To explore a prebuilt app that you can customize, create a :ref:`template 
-app <template-apps>`. Template applications are working apps that use Realm 
+app <template-apps>`. Template applications are working apps that use {+service-short+} 
 services to do things like:
 
 - Use Sync to synchronize data between mobile clients and the linked 
   MongoDB Atlas collection.
 - Host a Todo web app using the :ref:`GraphQL API <graphql-api>`
-- Host a Todo web app using the :ref:`Realm Web SDK <web-intro>`
+- Host a Todo web app using the :ref:`{+realm-web+} SDK <web-intro>`
 - Manage event-driven :ref:`Database Triggers <database-trigger>` 
   to update views in a separate collection
 
 Template apps are working apps you can run and change to experiment with 
-Realm. These apps are a good choice for developers who prefer to learn by 
+{+service-short+}. These apps are a good choice for developers who prefer to learn by 
 experimentation, versus the more description-heavy tutorials above.
 
 Summary
```

Comment:
Can't leave a note below, but on line 290/288, we say "build a {+app+}" - if we leave this source constant as "Atlas App Services" - the "a" should be "an". We should look for anywhere we've renamed to "Atlas" or "App" and make sure there isn't a preceding "a" - that should be swapped to "an."